src/scss/toolbar.scss

```diff
@@ -277,6 +276,7 @@
 }
 
 .toolbar-panels {
+  z-index: -1;
```

Comment:
The changes to how the Toolbar layering is setup are in a separate commit (https://github.com/bellingcat/ukraine-timemap/pull/48/commits/a136f00afc65980afe4feaa0ec14aef87fb2242d). The message there gives a detailed explanation of why this was necessary, please refer to it.

This can be merged into `translation` branch now. But I will make a pull request (to the `main` and hopefully upstream) with this and other refactorings before we get to merging `translation` into `main`. When the final merge is done I want it to only include translation-related features/additions, so I am paying special attention to keep all the changes that are made to enable those features/additions easy to pick out.